lib/ab/version.rb

```diff
@@ -1,3 +1,3 @@
 module Ab
-  VERSION = '0.2.4'
+  VERSION = '0.2.5'
```

Comment:
We bump versions in a separate commit:
https://github.com/vinted/ab/commits/master